examples/with-viem/src/createNewWallet.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  TurnkeyClient,
+  createActivityPoller,
+  TurnkeyActivityError,
+} from "@turnkey/http";
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import * as crypto from "crypto";
+import { refineNonNull } from "./util";
+
+export async function createNewWallet() {
+  console.log("creating a new wallet on Turnkey...\n");
+
+  const walletName = `ETH Wallet ${crypto.randomBytes(2).toString("hex")}`;
+
+  try {
+    const turnkeyClient = new TurnkeyClient(
+      { baseUrl: process.env.BASE_URL! },
+      new ApiKeyStamper({
+        apiPublicKey: process.env.API_PUBLIC_KEY!,
+        apiPrivateKey: process.env.API_PRIVATE_KEY!,
+      })
+    );
+
+    const activityPoller = createActivityPoller({
+      client: turnkeyClient,
+      requestFn: turnkeyClient.createWallet,
+    });
+
+    const completedActivity = await activityPoller({
+      type: "ACTIVITY_TYPE_CREATE_WALLET",
+      timestampMs: String(Date.now()),
+      organizationId: process.env.ORGANIZATION_ID!,
+      parameters: {
+        walletName,
+        accounts: [
+          {
+            curve: "CURVE_SECP256K1",
+            pathFormat: "PATH_FORMAT_BIP32",
+            path: "m/44'/60'/0'/0/0",
+            addressFormat: "ADDRESS_FORMAT_ETHEREUM",
+          },
```

Comment:
we should probably have SDK-level helpers for these constants at some point:
```ts
accounts: [EthereumAccountParam({"index": 0}), EthereumAccountParam({"index": 1}]
```
Would be pretty sweet!

Reply:
absolutely... the more we can abstract some of these details away, the better!